applications/luci-app-ddns/htdocs/luci-static/resources/view/ddns/overview.js

```diff
@@ -0,0 +1,993 @@
+			uci.load('ddns'),
+			this.callDDnsGetServicesStatus(),
+			callDDnsGetStatus(),
+			this.callDDnsGetEnv(),
+			L.resolveDefault(fs.read('/etc/ddns/services'), ''),
+			L.resolveDefault(fs.read('/etc/ddns/services_ipv6'), ''),
+		]);
+	},
+
+	render: function(data) {
+		var services = data[0];
+		var resolved = data[1];
+		var status = data[2];
+		var env = data[3];
+		var logdir = uci.get('ddns', 'global', 'ddns_logdir') || "/var/log/ddns";
+		
+		var services4 = []; 
+		var services6 = [];
+
+		data[4].split('\n').forEach(function(item) {
+			if (!item.startsWith("#")) {
+				services4.push(item.split('\t')[0].slice(1,-1));
+			}
+		});
+
+		data[5].split('\n').forEach(function(item) {
+			if (!item.startsWith("#")) {
+				services6.push(item.split('\t')[0].slice(1,-1));
+			}
+		});
+
+		var m, s, o;
+
+		m = new form.Map('ddns', _('Dynamic DNS'),);
+
+		var is = m.section(form.NamedSection, 'global', 'ddns', _('Information'));
+
+		s = is;
+
+		o = s.option(form.DummyValue, '_version', _('Dynamic DNS Version'));
+		o.cfgvalue = function() {
+			return status[this.option] || _('Error in gaining value')
+		};
+
+		o = s.option(form.DummyValue, '_enabled', _('State'));
+		o.cfgvalue = function() {
+			var res = status[this.option];
+			if (!res) {
+				this.description = _("Currently DDNS updates are not started at boot or on interface events.") + "<br>" +
+				_("This is the default if you run DDNS scripts by yourself (i.e. via cron with force_interval set to '0')")
+			}
+			return res ? _('DDNS Autostart enabled') : _('DDNS Autostart disabled')
+		};
+
+		o = s.option(form.DummyValue, '_toggle', '&nbsp');
+		o.cfgvalue = function() {
+			var action = status['_enabled'] ? 'stop' : 'start';
+			return E([], [
+				E('button', {
+				'class': 'cbi-button cbi-button-apply',
+				'click': L.ui.createHandlerFn(this, function() {
+					return Promise.all([callDDnsGetStatus()]).then(L.bind(function(data) {
+						var action = data[0]['_enabled'] ? 'stop' : 'start';
+						callInitAction('ddns', action == 'stop' ? 'disable' : 'enable');
+						return callInitAction('ddns', action);
+					}, this)).then(L.bind(m.render, m));
+				})
+			}, _(action.toUpperCase() + ' DDns'))]);
+		};
+
+		o = s.option(form.DummyValue, '_restart', '&nbsp');
+		o.cfgvalue = function() {
+			return E([], [
+				E('button', {
+				'class': 'cbi-button cbi-button-apply',
+				'click': L.ui.createHandlerFn(this, function() {
+					return callInitAction('ddns', 'restart').then(L.bind(m.render, m));
+				})
+			}, _('Restart DDns'))]);
+		};
+
+		// DDns hints
+
+		if (!env['has_ipv6']) {
+			o = s.option(form.DummyValue, '_no_ipv6');
+			o.rawhtml  = true;
+			o.title = '<b>' + _("IPv6 not supported") + '</b>';
+			o.cfgvalue = function() { return _("IPv6 is currently not (fully) supported by this system") + "<br>" +
+			_("Please follow the instructions on OpenWrt's homepage to enable IPv6 support") + "<br>" +
+			_("or update your system to the latest OpenWrt Release")};
```

Comment:
I am not sure if this is correct, but it depends on the original translations. If the original had the `<br>` inside the translation string, then you should do the same here to allow for reusing the existing translation strings. I haven't actually looked at the original code yet ,though.

Reply:
the original code have <br \> .... I still don't know if it's better to keep the old string OR change the existing string to use <br>